Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.14-alpine as builder
+
+RUN apk update && apk add --no-cache git ca-certificates && update-ca-certificates
+
+WORKDIR /go/src/github.com/scaleway/scaleway-cli
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+COPY cmd/ cmd/
+COPY internal/ internal/
+COPY .git .git/
+
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -a -ldflags "-w -extldflags -static -X main.GitCommit=$(git rev-parse --short HEAD) -X main.GitBranch=$(git symbolic-ref -q --short HEAD || echo HEAD) -X main.BuildDate=$(date -u '+%Y-%m-%dT%I:%M:%S%p')" ./cmd/scw
```

Comment:
can we reuse the build script somehow ? 

https://github.com/scaleway/scaleway-cli/blob/v2/scripts/build.sh

would that necessitate lot of refactor ?

Reply:
Hmm everything is possible :smile: But we'll need to refactor the script that's for sure

Reply:
done